values-all-in-one.yaml

```diff
@@ -248,6 +248,10 @@ clusterGroup:
       # timeout: 234
       # verbosity: "-v"
 
+  sharedValueFiles:
+    - '/cluster-overrides/values-{{ $.Values.global.clusterPlatform }}.yaml'
+    - '/cluster-overrides/values-{{ $.Values.global.clusterPlatform }}-{{ $.Values.global.clusterVersion }}.yaml'
```

Comment:
Maybe?

```suggestion
    - '/overrides/cluster/values-{{ $.Values.global.clusterPlatform }}.yaml'
    - '/overrides/cluster/values-{{ $.Values.global.clusterPlatform }}-{{ $.Values.global.clusterVersion }}.yaml'
```

Reply:
I also don't have an obvious winning proposal. I would avoid two folders for simplicity sake. Right now all our overrides are somehow cluster related so /cluster-overrides is not wrong. Maybe let's just call it `/overrides` for the time being and reassess later on?

Reply:
I think `/overrides` makes sense. The key thing is to get them out of `/`; like you say the "magic" variables are all cluster-wide. Let's do that.

Reply:
`/overrides/{appname}/...` for the random stuff?
or simply `/overrides/{appname}-{some global var}.yaml`

The best part of having them be explicit, is that ultimately the consumer can arrange them any way they like. What we're doing here is setting up the cargo-cult baseline.  Starting with `/overrides` seems fine for that